machines/issuersMachine.ts

```diff
@@ -232,19 +315,36 @@ export const IssuersMachine = model.createMachine(
     actions: {
       setIssuers: model.assign({
         issuers: (_, event) => event.data,
+      }),
+      setNoInternet: model.assign({
+        errorMessage: 'noInternetConnection',
+      }),
+      setDownloadingCreds: model.assign({
+        loadingReason: 'downloadingCredentials',
+      }),
+      setTokenLoadingReason: model.assign({
```

Comment:
instead of separate setter for Loading Reason context variable, can't we pass a argument and unify them
<img width="598" alt="image" src="https://github.com/mosip/inji/assets/94220135/c3c8e1f9-41eb-49d9-b36e-0ab84a95d232">

Reply:
You mean like inline actions? if so, Would inline actions make it hard to debug / visualize the machine?

Reply:
Renamed function accordingly